enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -499,6 +499,30 @@ Dashboards that support editing if a user has permissions for this, could be emb
   but without its cards - at this stage dashboard title, tabs and cards grid is rendered, but cards content is not yet
   loaded.
 
+### Creating a Question
+
+With the `NewQuestion` component, you can create a new question from scratch using the Metabase Notebook Editor. 
+
+#### Parameters
+
+No parameters are required for this component.
+
+```tsx
+import React from "react";
+import { MetabaseProvider, NewQuestion } from "@metabase/embedding-sdk-react";
+
+const config = {...}
+
+export default function App() {
+  return (
+    <MetabaseProvider config={config}>
+        <NewQuestion />
```

Comment:
nit: We should discuss and align with the name with this PR https://github.com/metabase/metabase/pull/46802. As that one introduce a way to create a dashboard which is comparable to this PR, and I expect components from both PR to have a similar name pattern.

Reply:
let's do that when ModifyQuestion is in the works